builtin/providers/github/resource_github_team.go

```diff
@@ -50,6 +55,17 @@ func resourceGithubTeamCreate(d *schema.ResourceData, meta interface{}) error {
 	if err != nil {
 		return err
 	}
+
+	if ldap_dn := d.Get("ldap_dn").(string); ldap_dn != "" {
```

Comment:
I think if the design of this is an update call, then we should maybe have Create -> Update -> Read

That way, we won't have the same logic in both Create and Update

Thoughts?